libsrc/common/strnlen.s

```diff
@@ -0,0 +1,54 @@
+; size_t __fastcall__ strnlen (const char* s, size_t maxlen);
+
+.export     _strnlen
+.import     popax
+.importzp   ptr1, tmp1, tmp2, tmp3, tmp4
+
+.proc _strnlen
+        ; Fetch string pointer.
+        sta     ptr1
+        stx     ptr1+1
```

Comment:
You have the parameters backwards.  This is the _last_ one, `maxlen`.

Reply:
Oh, damn! Was under the impression that this code was already in use at my place. Sorry.